qiskit/aqua/algorithms/classifiers/vqc.py

```diff
@@ -269,10 +269,11 @@ def batch_data(self, data, labels=None, minibatch_size=-1):
         if 0 < minibatch_size < len(data):
             batch_size = min(minibatch_size, len(data))
             if labels is not None:
-                shuffled_samples, shuffled_labels = shuffle(data, labels, random_state=self.random)
+                shuffled_samples, shuffled_labels = shuffle(data, labels,
+                                                            random_state=aqua_globals.random_seed)
                 label_batches = np.array_split(shuffled_labels, batch_size)
             else:
-                shuffled_samples = shuffle(data, random_state=self.random)
+                shuffled_samples = shuffle(data, random_state=aqua_globals.random_seed)
```

Comment:
Just to this understand correctly: Before the random generator itself was passed, now you're passing the random seed. If the random seed is None, then `sklearn.utils.shuffle` uses a numpy random as seed. But now as additional feature we have better reproducibility since we can set the seed. Right?

Reply:
the shuffle method doesn't accept a Generator, only RandomSate. Since we don't have a RandomState anymore I could only pass the seed. That was the reason for this change.